docs/configuration.rst

```diff
@@ -219,7 +219,8 @@ try to use ``S3Hook('MyS3Conn')``.
 In the Airflow Web UI, local logs take precedance over remote logs. If local logs
 can not be found or accessed, the remote logs will be displayed. Note that logs
 are only sent to remote storage once a task completes (including failure). In other
-words, remote logs for running tasks are unavailable.
+words, remote logs for running tasks are unavailable. Logs are stored in the log
+folder as ``dag_id/task_id/execution_date/try_number.log``.
```

Comment:
maybe add some angle brackets so it is clear they are templated variables, itwill read a bit better